src/pages/register.tsx

```diff
@@ -0,0 +1,211 @@
+import Link from 'next/link';
+import Image from 'next/image';
+
+const Register = () => {
+    const inputStyle =
+        'w-full bg-input-gray border-input-gray rounded-full p-2 px-3 mb-4';
```

Comment:
It would be better to add component styles to the `src/styles/globals.css file.` You can do so like this:
```css
@layer components
{
  .input {
    @apply w-full bg-input-gray border-input-gray rounded-full p-2 px-3 mb-4;
  }
}
``